rclcpp/include/rclcpp/node.hpp

```diff
@@ -109,19 +127,27 @@ class Node
     rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
   /* Create and return a Service. */
-  template<typename ServiceT>
-  typename rclcpp::service::Service<ServiceT>::SharedPtr
+  template<typename ServiceT, typename FunctorT>
+  typename function_arity<
+    FunctorT,
+    2,
+    typename rclcpp::service::Service<ServiceT>::SharedPtr
+    >::type
```

Comment:
The indentation looks difficult to "parse". Would the following be clearer?

      typename function_arity
      <
        FunctorT,
        2,
        typename rclcpp::service::Service<ServiceT>::SharedPtr
      >::type